PerfNext/public/lib/js/util.js

```diff
@@ -640,28 +640,32 @@ function insertCapabilityEnvVars(capabilityName, hwEnvClassName, idToFind) {
 	
 	var hostNode = getHostNode() ;
 	
-	//Will match all ids starting with the capabilityName
-	var capabilityNode = hostNode.querySelector('[name^="' + capabilityName + '"]');
+	var capabilities = hostNode.querySelectorAll('capability');
+	
 	var envVARs = 0;
-	if (capabilityNode)
-	{
-		envVARs = capabilityNode.children;
-		console.log("util.js insertCapabilityEnvVars capabilityNode envVARs.length: "+envVARs.length); 
-	}
 	
+	//
+	for (const capability of capabilities) {
+		// Ignore any lowercase capabilities as well as benchmark variant. Only the name is relevant example: 'LIBERTY', 'SPECJBB' 
+		// Example: LibertyDayTrader3 would be matched to capability 'LIBERTY' as hardware variables do not depend on variant of benchmark being run
+		if (capability.getAttribute('name') && capabilityName.toUpperCase().indexOf(capability.getAttribute('name')) !== -1){ 
```

Comment:
Maybe have a better name since it's easy to confuse `capability` with `capabilityName`.